### PR TITLE
Add sistent github issue template

### PR DESCRIPTION
**Pull request by sudhanshutech**
**Description**

This PR adds a new github issue templated for sistent which can be used for purpose like to add any component or file any issue while using the component.
Sistent repo link is also mentioned below as resources as help

**Notes for Reviewers**

**[Signed commits](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Comment by l5io** [comment]
🚀 Preview for commit 7cb92c3657c614cfbf1644e4a72c30cd90a91ef7 at: https://66ed54cf2037df3f9082b334--layer5.netlify.app

File: .github/ISSUE_TEMPLATE/sistent_feature.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+<!-- Add any other context or screenshots about the feature request here. -->
+
+### Implementation
+<!-- Specifics on the approach to fulfilling the feature request. -->
+
+---
+<img src="https://raw.githubusercontent.com/layer5io/layer5/master/.github/assets/images/layer5/5-light-small.svg" width="24px" align="left" /><h2>Contributor Resources and <a href="https://layer5.io/community/handbook">Handbook</a></h2>
+
+The layer5.io website uses Gatsby, React, and GitHub Pages. Site content is found under the [`master` branch](https://github.com/layer5io/layer5/tree/master).
+- 🎨 See [Sistent Repository](https://github.com/layer5io/sistent).
```
Comment (leecalcote) [inline .github/ISSUE_TEMPLATE/sistent_feature.md:24]: Good

**Review by leecalcote — APPROVED** [review]
Looking good. 👍

**Comment by vishalvivekm** [comment]
@sudhanshutech Thank you for your contribution!
Let's discuss this during the website call on Monday at 5:30 PM IST (7:00 AM CT).

Please add it as an agenda item to the [meeting minutes](https://docs.google.com/document/d/1XczAHXVe2FIWPqiF57ospJ43zw5cZQ7ui8mn39v5EvA/edit?usp=drivesdk).

**Pull request merged**